frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -58,8 +58,9 @@ body:where(:global(.mb-wrapper)) {
   --mb-color-background: var(--mb-color-bg-black);
   --mb-color-background-selected: var(--mb-base-color-brand-60);
   --mb-color-background-hover: var(--mb-base-color-brand-70);
-  --mb-color-background-disabled: var(--mb-base-color-gray-70);
+  --mb-color-background-disabled: var(--mb-base-color-orion-70);
   --mb-color-background-brand: var(--mb-base-color-brand-60);
+  --mb-color-switch-off: var(--mb-base-color-orion-70);
```

Comment:
added a new color for [toggle off](https://carbondesignsystem.com/elements/color/tokens/#miscellaneous)